### PR TITLE
Seperating Sections

### PR DESCRIPTION
**Pull request by IHateYourCode**
Hi, first of all I would like to **thank you for creating this project**,
like you I'm also not a cryptographer, however I don't enjoy being
forced to use encryption/... everywhere.

Well obviously if you don't, you will have a lot of problems..

Of course I don't inherently dislike cryptography or anything,
in fact, the state of most services / data holders / .. scares
the everliving shit out of me.

Which is why as a someone who wants to make things more **easy**,
**obvious** and **understandable**, I want to contribute to your project,
so more people can get a better understanding of what they **want**
to be using for their projects.

---

**Changes of the pull request:**
Basically just simple separation of
the sections as described in the
individual commits.

### CODE REVIEW

**Comment by samuel-lucas6** [comment]
Hi @IHateYourCode,

Thanks for the support and going through the trouble of doing this PR, but I'm afraid I'd rather keep the guidelines as one file for three reasons:

1. The README already has a contents menu if you use the three little dots/lines next to 'README.md', which allows someone to jump to a certain section relatively easily.
2. The guidelines are accessible via GitHub Pages [here](https://samuellucas.com/Cryptography-Guidelines/) as if it's a blog post, and the original plan was for it to be a blog post on my website. The reason I opted for GitHub was mainly for the collaboration aspect.
3. I personally prefer having everything in one file so that you can scroll between the sections easily, and all of the other guidelines I've seen on GitHub are just one file.

Therefore, I'm going to close this PR. I'm sorry that it won't be merged.

**Comment by IHateYourCode** [comment]
I can't say that the contents menu is very useful,
it listing every sub headline is quite cluttering.
Github Pages works with multiple files btw, not
every blog consists of just one page and one
doesn't have to follow the design of every
other guideline.

As for your preferences, I can't say I'm of the
side, which is why, as a compromise, I'll just have
to continue this as a separate version I guess.

Well thank goodness this is a topic that won't
radically change from one day to the other, so
updating changes from your main repository
won't be too much of a challenge I hope.

Thank you for your consideration, I guess.

**Comment by samuel-lucas6** [comment]
> I can't say that the contents menu is very useful, it listing every sub headline is quite cluttering.

I understand your point. Perhaps it's worth adding a separate contents section at the top of the document.

> Github Pages works with multiple files btw, not every blog consists of just one page and one doesn't have to follow the design of every other guideline.

I'm aware, but blog posts are one page. This isn't meant to be a website, and it's more of a hassle switching files to read different sections in my opinion.

> As for your preferences, I can't say I'm of the side, which is why, as a compromise, I'll just have to continue this as a separate version I guess.
> 
> Well thank goodness this is a topic that won't radically change from one day to the other, so updating changes from your main repository won't be too much of a challenge I hope.

That's fine. I do intend to add some things at some point, but I'm becoming rather busy. Other than that and perhaps correcting some Markdown formatting mistakes I've missed, not much should change for a while. For example, it'll be a long time before any post-quantum algorithms and CAESAR final portfolio algorithms will be recommended.
